build_for_ultimaker.sh

```diff
@@ -9,7 +9,7 @@ ARCH="${ARCH:-armhf}"
 ARM_EMU_BIN=
 CUR_DIR=$(pwd)
 BUILD_DIR="${CUR_DIR}/.build_${ARCH}"
-ROOTFS_IMG="rootfs.xz.img"
+TOOLBOX_IMAGE="um-update_toolbox.xz.img"
```

Comment:
i didn't realize this until now, but this is really ugly, we now have the name in 2 places and they have to match.

In that case, we want to do the proposed above change, and pass this variable to the test script along, so we don't have to keep these names in sync.

Reply:
Yes it is, it is even more difficult because the init.sh script is also using the image name. In the future we should have some digital signing and get rid of the fixed names.